addons/longer-wiwo/userscript.js

```diff
@@ -0,0 +1,23 @@
+export default async function ({ msg }) {
+  const status = $('textarea[name="status"]');
+  const left = $($("#status-chars-left")[0].parentElement);
+
+  // remove scratch's event listeners
+  status.off("input");
+  status.off("focusin");
+  status.off("focusout");
```

Comment:
This seems to work, but if you're removing the event listeners, why does it still upload the updated WIWO when its changed? What am I missing? I want to know what's going on lol

Reply:
Also, I noticed that when the WIWO does update, Scratch adds a green modal to show it saved, but for some reason that modal is empty (has no text). Any ideas why?

Reply:
1: that event listener is on a different element, `div#status`
2: idk, i saw that before i even started working on this

Reply:
> Also, I noticed that when the WIWO does update, Scratch adds a green modal to show it saved, but for some reason that modal is empty (has no text). Any ideas why?

This is a Scratch bug (also happens without Scratch Addons). It's supposed to say "Changes saved".